src/styles/theme.scss

```diff
@@ -337,22 +411,40 @@ h5 {
   background-color: $primary-colour;
 }
 
+.card--alternate.card--with-icon {
+  background-color: transparent;
+  color: $secondary-colour;
+  border: 2px solid $icon-background;
+  border-radius: 10px;
+  article h2 {
+    color: $text-colour;
+  }
+
+}
+
 /* ------------------------------
 
     Info to help you
 
 ------------------------------ */
 
+.info {
+  background-color: transparent;
```

Comment:
Would these be covered if you took the `casrd--with-icon` selector off the above?